### DISCUSSION TITLE
1.5.1 instal fail

### DISCUSSION THREAD

**Discussion by szl-926929** [Kong/kong#7447] [Help] [answered]
### Summary
- Kong version：1.5.1
wget https://download.konghq.com/gateway-1.x-centos-7/Packages/k/kong-1.5.1.el7.amd64.rpm
yum install kong-1.5.1.el7.amd64.rpm
 /usr/local/bin/kong start

Error:
/usr/local/share/lua/5.1/kong/cmd/start.lua:75: /usr/local/share/lua/5.1/kong/cmd/start.lua:64: nginx: [warn] [lua] getssl.lua:10: OpenResty patch missing. See https://github.com/Kong/openresty-patches
nginx: [error] init_by_lua error: /usr/local/share/lua/5.1/kong/init.lua:361: ssl_cert: no such file at /usr/local/kong/ssl/kong-default.crt,/usr/local/kong/ssl/kong-default-ecdsa.crt
stack traceback:
	[C]: in function 'assert'
	/usr/local/share/lua/5.1/kong/init.lua:361: in function 'init'
	init_by_lua:3: in main chunk

stack traceback:
	[C]: in function 'error'
	/usr/local/share/lua/5.1/kong/cmd/start.lua:75: in function 'cmd_exec'
	/usr/local/share/lua/5.1/kong/cmd/init.lua:87: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:87>
	[C]: in function 'xpcall'
	/usr/local/share/lua/5.1/kong/cmd/init.lua:87: in function </usr/local/share/lua/5.1/kong/cmd/init.lua:44>
	/usr/local/bin/kong:9: in function 'file_gen'
	init_worker_by_lua:47: in function <init_worker_by_lua:45>
	[C]: in function 'xpcall'
	init_worker_by_lua:54: in function <init_worker_by_lua:52>

**Comment by locao** [comment, marked as answer]
It seems you have another Nginx/OpenResty already installed on this environment. Please, paste the output of the command: 
```$ /usr/local/bin/kong start --vv```

**Reply by dndx** [reply]
Also I would like to mention that 1.5 is a really old version of Kong and is no longer supported by us, please use the latest 2.x series instead.
